lte/gateway/python/integ_tests/s1aptests/test_attach_detach_no_ueContext_release_comp.py

```diff
@@ -18,15 +18,15 @@
 import s1ap_wrapper
 
 
-class TestAttachDetachNoUeContextReleseComp(unittest.TestCase):
+class TestAttachDetachNoUeContextReleaseComp(unittest.TestCase):
 
     def setUp(self):
         self._s1ap_wrapper = s1ap_wrapper.TestWrapper()
 
     def tearDown(self):
         self._s1ap_wrapper.cleanup()
 
-    def test_attach_detach_no_ueContRelComp(self):
+    def test_attach_detach_no_ueContext_release_Comp(self):
```

Comment:
Either you can rename as "test_attach_detach_no_ue_context_release_comp" or "test_attach_detach_no_uecontext_release_comp"